RULES.md

```diff
@@ -1,62 +1,118 @@
+
+### Top-level fields
 #### expires
-When this configuration file expires. Can be in epoch seconds or in UTC.
+When this configuration file expires, in UTC. Can be in epoch seconds from 00:00:00 UTC on 1 January 1970, or a string `yyyy-MM-dd'T'HH:mm:ssZZZZ`. If the file has ceased being regularly updated for any reason, and the policy file has expired, the MTA should fall-back to opportunistic TLS for e-mail delivery, and the system operator should be alerted.
 
 #### timestamp
-When this configuration file was distributed/fetched. Can be in epoch seconds or UTC.
+When this configuration file was distributed/fetched, in UTC. Can be in epoch seconds from 00:00:00 UTC on 1 January 1970, or a string `yyyy-MM-dd'T'HH:mm:ssZZZZ`. This field will be monotonically increasing for every update to the policy file. When updating this policy file, should validate that the timestamp is greater than or equal to the existing one.
+
+#### policy-aliases
+A mapping of alias names onto a policy. Domains in the `policies` field can refer to policies defined in this object.
 
-#### tls-policies
-A mapping of domains to policies.
+#### policies
+A mapping of mail domains (the part of an address after the "@") onto a "policy". Matching of mail domains is on an exact-match basis. For instance, `eff.org` would be listed separately from `lists.eff.org`. Fields in this policy specify security requirements that should be applied when connecting to any MTA server for a given mail domain. If the `mode` is `testing`, then the sender should not stop mail delivery on policy failures, but should produce logging information.
 
-## Policy fields
+#### pinsets
+Maps a unique pinset name to a list of accepted pins. For a given pinset, a certificate is accepted if at least one of the `static_spki_hashes` SPKIs is found in the chain.
+
+### Pinset fields
+#### static-spki-hashes
+The set of allowed SPKIs hashes.
+
+### Policy fields
 Every field in `tls-policies` maps to a policy object. A policy object can have the following fields:
 
+#### policy-alias
+
+If set, other fields are ignored. This value should be a key in the upper-level "policy-aliases" configuration object. The policy for this domain will be configured as the denoted policy in the "policy-aliases" object.
+
 #### min-tls-version
 Default: `TLSv1.2`
 
-The minimum TLS version expected from this domain.
+The minimum TLS version expected from this domain. Sending mail to domains under this policy should fail if the sending MTA cannot negotiate a TLS version equal to or greater than the listed version. Valid values are `TLSv1`, `TLSv1.1`, and `TLSv1.2`.
 
 #### mode
-Default: `testing`
+Default: `testing` (required)
 
 Either `testing` or `enforce`. If `testing` is set, then any failure in TLS negotiation is logged and reported, but the message is sent over the insecure communication.
```

Comment:
Here you use `testing` or `enforce`, but in the example given earlier one of the entries is `log-only`. Also, should we have `none` as an option to, to indicate a domain is stopping using STARTTLS (a la how MTA-STS does)?

Reply:
Thanks for the `log-only` catch-- I missed that!
Re: `none`, I thought about this a bit more. People should be able to set `mode: none` on their MTA-STS to opt-out of the list, but I'm not sure there's value to updating someone's entry to `mode: none` vs. simply removing the policy from the index.

Reply:
Agreed--`mode: none` is useful in MTA-STS because the policy delivery channel is authenticated (by reading DNS and HTTPS). Do we want to talk at all about the signature verification part of distributing the policy rules for STARTTLS Everywhere in `RULES.md`? Or at least make it more formal somewhere? (I don't think that has to hold up this PR--just thought I'd bring it up.)

Reply:
There is a brief mention of it in `RULES.md` that refers to an verifying against a PGP key, but it should definitely be more specific. I made an issue (#44) for this to keep it in mind once we finalize the process on our end (and if we want to continue discussing it elsewhere).